packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -831,6 +868,24 @@ class PageTransitionsTheme with Diagnosticable {
     );
   }
 
+  /// Provides the delegate transition for the target platform.
+  ///
+  /// {@macro flutter.widgets.delegatedTransition}
+  DelegatedTransition? delegatedTransition(TargetPlatform platform, bool allowSnapshotting) {
+    final PageTransitionsBuilder matchingBuilder =
+      builders[platform] ?? const ZoomPageTransitionsBuilder();
```

Comment:
Does this mean that every page transition in PageTransitionBuilder.builders will now have a delegatedTransition, which will be itself by default? Even user-defined transitions?

By default, if ThemeData.pageTransitionTheme is not passed, does it create one or does this all get bypassed?

Reply:
> Does this mean that every page transition in PageTransitionBuilder.builders will now have a delegatedTransition, which will be itself by default?

Right now only Zoom will, the rest will be null, which means we'll have the existing behavior before this PR. We could add delegates to the rest of them, but I think that should be in a later PR.

> By default, if ThemeData.pageTransitionTheme is not passed, does it create one or does this all get bypassed?

If there's no ThemeData.pageTransitionTheme provided, it will use the default one, which uses a Zoom transition everywhere but iOS, which uses the Cupertino transition. Which both have delegates.